docs/pricing.rst

```diff
@@ -1,32 +1,33 @@
 Pricing
 =======
 
-All paying customers receive unlimited members and projects, email support, and access to all of Sentry’s product features. Pricing is based on your data volume and has two options. You may either pay for what you use without long term commitments or prepay for significant discounts (up to 70%).
+All paying customers receive unlimited members and projects, email support, and access to every Sentry product feature for a base price of $15 monthly (or $144 annually). Pricing is based on your event data volume and has two options. You may either pay for what you use with monthly on-demand capacity or prepay for significant discounts (up to 70%) with reserved capacity.
 
 Concepts
 --------
 
 Events
 ~~~~~~
 
-An event is a crash report (also known as exception or error) that is sent to Sentry’s server to be processes. Additional metadata around an event (breadcrumbs, user context, environment, etc) do not count as separate events.
+An event is a crash report (also known as an exception or error) that's sent to Sentry’s server for processing. Additional metadata around an event (breadcrumbs, user context, environment, etc) do not count as separate events.
 
 Capacity
 ~~~~~~~~
 
-Capacity is the total number of events sent by and billed to your organization. Capacity comes in two types: on-demand and reserved.
+Capacity is the total number of events sent by and billed to your organization. You can purchase capacity in two ways: on-demand and reserved.
 
 On-demand Capacity
 ``````````````````
 
-On-demand capacity is billed at the end of each billing cycle. You are charged per processed event, rounded up to the nearest cent.
+On-demand capacity is billed at the end of each billing cycle. You are charged per processed event, rounded up to the nearest cent. You have the ability to set on-demand caps so you can stay on budget. See _`Billing Controls <https://docs.sentry.io/pricing/#billing-controls>`_ for more information. 
```

Comment:
With ``:ref:``:

http://www.sphinx-doc.org/en/1.5.1/markup/inline.html#cross-referencing-arbitrary-locations

And links something like this (but using whatever their docs say):

```
:doc:`Raven.js documentation <../../../clients/javascript/index>`
```